.github/workflows/e2e.yml

```diff
@@ -0,0 +1,69 @@
+name: E2E
+
+on:
+  # TODO: do this on push to master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js 10.16.3
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.16.3
+      - name: Build the Drone-simulator image
+        id: drone
+        run: |
+          cd packages/drone-simulator
+          docker build . --iidfile TEMP_DRONE_IMAGE_ID.txt
+          DRONE_DOCKER_IMAGE_ID=$(cat TEMP_DRONE_IMAGE_ID.txt)
+          echo ::set-env name=DRONE_DOCKER_IMAGE_ID::$(echo $DRONE_DOCKER_IMAGE_ID)
+      - name: Build the API image
+        id: api
+        run: |
+          cd packages/api
+          docker build . --iidfile TEMP_API_IMAGE_ID.txt
+          API_DOCKER_IMAGE_ID=$(cat TEMP_API_IMAGE_ID.txt)
+          echo ::set-env name=API_DOCKER_IMAGE_ID::$(echo $API_DOCKER_IMAGE_ID)
+      - name: Build the Client image
+        id: client
+        run: |
+          cd packages/client
+          docker build . --build-arg MAPBOX_TOKEN --build-arg GRAPHQL_API --build-arg GRAPHQL_WS_URI --iidfile TEMP_CLIENT_IMAGE_ID.txt
+          CLIENT_DOCKER_IMAGE_ID=$(cat TEMP_CLIENT_IMAGE_ID.txt)
+          echo ::set-env name=CLIENT_DOCKER_IMAGE_ID::$(echo $CLIENT_DOCKER_IMAGE_ID)
+      - name: Start all services
+        run: |
+          cd packages/api
+          docker-compose -f docker-compose-ci.yml up -d
+      - name: Bootstrap dependencies
+        run: |
+          npm install
+          npm run bootstrap
+      - name: Run Cypress-tests
+        run: |
+          cd packages/client
+          npm install
```

Comment:
Do we need to install dependencies again here? I see that npm run cypress:ci is running "npx cypress run". Maybe im missing something here but seems redundant.

Reply:
yep, it's a leftover, will remove if all passes :D